styles/cart.css

```diff
@@ -0,0 +1,82 @@
+.cart-heading{
+    border-bottom: solid var(--primary-clr) 2px;
+    color:var(--accent-clr);
+    margin:1rem 3rem;
+    padding:1rem;
+}
+
+.cart-item-purple, .cart-item-pink{
+    background-color:var(--secondary-clr);
+    border-radius:1rem;
+    box-shadow: rgba(99, 99, 99, 0.2) 0px 2px 8px 0px;
+    margin:1.5rem 0;
+    padding:1rem;
+    width:50rem;
+}
+
+.cart-item-pink{
+    background-color: var(--teritiary-clr);
+}
+
+.cart-item-img{
+    width:12rem;border-radius:1rem;
+}
+
+.card-offer-percent{
+    color:var(--primary-clr);
+}
+
+.qty-btn{
+    width:2rem;
+    height:2rem;
+    margin:5px;
+    background-color:var(--clr-white);
+    border-radius: 50%;
+    border:none;
+    font-weight: bold;
+    font-size:larger
+}
+
+.cart-card-dismiss{
+    background: transparent;
+    border:none;
+    width:2rem;
+    height:2rem;
+    font-size: medium;
+}
+
+.cart-card-dismiss:hover{
+    border:none;
+    border-radius:50%;
+    background-color: var(--secondary-clr);
+}
```

Comment:
This hover is not working for purple-colored cards.

Reply:
Since the color on hover is same as the card color it's not being shown. Will change that to a contrast color. Thanks.